### DISCUSSION TITLE
UMinho title page and spine

### DISCUSSION THREAD

**Discussion by Marcelo-Alves7** [joaomlourenco/novathesis#427] [Q&A]
Hello professor,

I am preparing the final versions of my thesis. I'm using branch 7.0.0 in Overleaf.

My first doubt concerns the spine, since despite my title being in 3 lines, the spine should only appear in two lines, as in the various examples provided by UMinho. How can I change the spine just to have the title on 2 lines (like the example)?
![image](https://user-images.githubusercontent.com/98588105/230624428-a173d1fe-b591-4095-bf6b-7861350db0d7.png)

In my case it looks like this:
![image](https://user-images.githubusercontent.com/98588105/230624513-5f454b1b-7285-4f93-ad5f-bc343f513456.png)

The other question concerns the title page since the type and size of the font must be the same for "Doctorate thesis", "Name of the doctorate" and "Work developed under the supervision of:", however in the template, the last piece of text appears different from the rest. How can I change (to match the example)?
![image](https://user-images.githubusercontent.com/98588105/230624334-7b0b0cf9-de76-4adf-92b2-388dc7e4d089.png)

In my case it looks like this:
![image](https://user-images.githubusercontent.com/98588105/230624680-5ea5016f-c07b-4e9e-9a1f-3b39abc6b673.png)

I appreciate your help!

**Comment by joaomlourenco** [comment]
@Marcelo-Alves7 

Answer to question 1: “Configure title in spine”.
In file `Config/5_packages.tex` add
```latex
\spine(title):={Your title with \\ to break the line.}
```

Answer to questions 2: “Fixing the fonts in the second cover page”
Replace the file `NOVAthesisFiles/Schools/uminho/uminho-defaults.ldf`with the new one from the repository (7.0.0 branch).

**Reply by Marcelo-Alves7** [reply]
Professor @joaomlourenco,

Regarding your answer to question 1, it does not produce any effect.
Apparently, the title on the spine seems to "copy" the main title and doesn't allow customization.

![image](https://user-images.githubusercontent.com/98588105/230876695-941bd3d8-a903-4d47-a97c-1efc6c37714b.png)

As you can see, the title on the spine copies the main title in its entirety.
It has to be adjustable on the spine, as it should only appear in two lines.

Thanks

**Reply by Marcelo-Alves7** [reply]
Professor @joaomlourenco,

I think it makes sense to comment out the line in `NOVAthesisFiles/Schools/uminho/cover-final.tex`
`\spine(title):={\thedoctitle(\option{/novathesis/lang/cover},main,cover)}`

If you do that, it already allows me to customize according to what you suggested: 
`\spine(title):={Your title with \\ to break the line.}` in `Config/5_packages.tex`

However, even so, putting my title without line breaks, doesn't the text have room to stay in just two lines?
![image](https://user-images.githubusercontent.com/98588105/230879433-844ee2cb-8045-478f-a7a8-b02f99e86881.png)

What do you think?
Is it possible to slightly enlarge the text box for the title in the spine?
Or maybe reduce the font size a bit in the spine title?
